Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM golang:1.15.2 AS builder
+
+COPY . /build
+
+RUN cd /build && \
+    CGO_ENABLED=1 GOOS=linux GOARCH=amd64 go build -o app .
+
+# Real image
+FROM ubuntu:focal
```

Comment:
I am using Ubuntu as alpine doesn't have the libs to run the binary and distroless isn't multiarch

Reply:
The missing libs is because you were building it with CGO and missing some ldflags. 

Using `GO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -ldflags="-w -s" -o bmwcd_exporter .` will build the binary without relying on anything from the build host.

Reply:
It is possible to use the `scratch` image, but we'll need to copy at least the certificates from the golang image:
```
COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
```
otherwise it's not possible to make a https request.